core/src/main/java/com/gentics/mesh/core/endpoint/node/BinaryFieldResponseHandler.java

```diff
@@ -54,20 +56,15 @@ public BinaryFieldResponseHandler(ImageManipulator imageManipulator, BinaryStora
 	 */
 	public void handle(RoutingContext rc, BinaryGraphField binaryField) {
 		rc.response().putHeader(HttpHeaders.ACCEPT_RANGES, "bytes");
-		if (!storage.exists(binaryField)) {
-			rc.fail(error(NOT_FOUND, "node_error_binary_data_not_found"));
+		if (checkETag(rc, binaryField)) {
```

Comment:
Check if there is a test which tests behaviour for missing binary files. This can happen due to nfs issues / rsync chaos.

Reply:
Please write test for binary transform endpoint